File: WordPress/Classes/ViewRelated/Notifications/Views/NoteBlockQuoteTableViewCell.m
#import "NoteBlockQuoteTableViewCell.h"
#import "WPStyleGuide+Notifications.h"



#pragma mark ====================================================================================
#pragma mark NoteBlockTextTableViewCell
#pragma mark ====================================================================================

@implementation NoteBlockQuoteTableViewCell

- (NSInteger)numberOfLines
{
    NSInteger const NoteBlockLabelMaxLines = 4;
    return NoteBlockLabelMaxLines;
}

- (CGFloat)labelPreferredMaxLayoutWidth
{
    UIEdgeInsets const NoteBlockLabelInsets = {0.0f, 46.0f, 0.0f, 20.0f};
    return CGRectGetWidth(self.bounds) - NoteBlockLabelInsets.left - NoteBlockLabelInsets.right;
}

@end

File: WordPress/Classes/ViewRelated/Notifications/Views/NoteBlockQuoteTableViewCell.swift
import Foundation


@objc public class NoteBlockQuoteTableViewCell : NoteBlockTextTableViewCell
{
    public override var numberOfLines: Int {
        return lines
    }

    public override var labelInsets: UIEdgeInsets {
        return insets
    }
    
    private let lines   = 0
    private let insets  = UIEdgeInsets(top:0, left:46, bottom:0, right:20)
}
